packages/api-db-mongodb/src/migration.ts

```diff
@@ -702,6 +709,73 @@ export const Migrations: Migration[] = [
         ]
       )
     }
+  },
+  {
+    // change embed block properties width and height from number to string
+    version: 18,
+    async migrate(db) {
+      const articles = db.collection(CollectionName.Articles)
+      const migrationArticles = await articles.find().toArray()
+
+      for (const article of migrationArticles) {
+        if (article.draft) {
+          article.draft.blocks.map((block: ArticleBlock) => {
+            if (block.type === BlockType.Embed) {
+              block.height = String(block.height)
+              block.width = String(block.width)
+            }
+          })
+        }
+        if (article.published) {
+          article.published.blocks.map((block: ArticleBlock) => {
+            if (block.type === BlockType.Embed) {
+              block.height = String(block.height)
+              block.width = String(block.width)
+            }
+          })
+        }
+        if (article.pending) {
+          article.pending.blocks.map((block: ArticleBlock) => {
+            if (block.type === BlockType.Embed) {
+              block.height = String(block.height)
+              block.width = String(block.width)
+            }
+          })
+        }
+        await articles.findOneAndReplace({_id: article._id}, article)
+      }
+
+      const pages = db.collection(CollectionName.Articles)
```

Comment:
This should be `CollectionName.Pages` right?

Reply:
Whooops yes :+1: